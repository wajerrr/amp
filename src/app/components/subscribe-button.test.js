import React from 'react';
import renderer from 'react-test-renderer';
import SubscribeButton from './subscribe-button';

describe('SubscribeButton', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<SubscribeButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
