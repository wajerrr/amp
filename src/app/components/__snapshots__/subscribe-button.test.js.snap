// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscribeButton should match snapshot 1`] = `
<a
  className="css-vwbibu"
  href="https://econ.st/2bs1gXz"
  role="button"
>
  Subscribe
</a>
`;
